ecl/hql/hqlgram2.cpp

```diff
@@ -192,6 +192,14 @@ bool attribute::isZero() const
     return (value && (value->getIntValue() == 0));
 }
 
+bool attribute::getBoolValue()
+{
+    OwnedHqlExpr expr = getExpr();
```

Comment:
This was only used for optOpt which (as per your recommendation below) I have changed to not use an expression - so this is moot and I will remove the function.